pages/activity/entries/titles.js

```diff
@@ -0,0 +1,26 @@
+import { StyledContainer } from "@/components/StyledContainer/StyledContainer";
+import StyledReturn from "@/components/StyledReturn/StyledReturn";
+import Link from "next/link";
+import { Wrapper } from "@/components/Wrapper/Wrapper";
+export default function TitleCards({ activities }) {
+  return (
+    <>
+      <StyledReturn />
+
+      <h1> Your Titles</h1>
+      {activities.map((activity) => {
+        return (
+          <>
+            <Wrapper>
+              <Link href={`/activity/${activity.id}`}>
+                <StyledContainer>
+                  <p>{activity.title}</p>
+                </StyledContainer>
+              </Link>
+            </Wrapper>
+          </>
```

Comment:
you don't need fragments here and you are missing the key prop:

```suggestion
            <Wrapper key={activity.id}>
              <Link href={`/activity/${activity.id}`}>
                <StyledContainer>
                  <p>{activity.title}</p>
                </StyledContainer>
              </Link>
            </Wrapper>
```